tutorial/20_recipes/002_multi_objective.py

```diff
@@ -0,0 +1,71 @@
+"""
+.. _multi_objective_lgbm:
+
+Multi-objective optimization example with LightGBM
```

Comment:
A nit (for consistency).
```suggestion
Multi-objective Optimization Example with LightGBM
```